packages/shared/src/enums.ts

```diff
@@ -125,6 +141,12 @@ export const chainConstants = {
     contractId: 4,
     blockTimeSeconds: 0.26,
   },
+  [Chains.Solana]: {
+    assets: [Assets.SOL, Assets.USDC],
+    gasAsset: Assets.SOL,
+    contractId: 5,
+    blockTimeSeconds: (400 + 800) / 2 / 1000,
```

Comment:
how is this calculated? Do you have some reference by any chhance?

Reply:
No I just read that it takes between 400ms and 800ms for a block, so i took the average